packages/react-button/src/components/Button/Button.types.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why do we need this here?

Reply:
removed.

Reply:
it was because of size_small, etc. Moved to per line so it's more scoped and obvious.